src/vos/tests/evt_ctl.c

```diff
@@ -750,6 +751,338 @@ global_teardown(void **state)
 #define NUM_PARTIAL 11
 #define NUM_EXTENTS 30
 
+/* create_expected_data:
+* Input values:  iter_flag, evt_data
+* Input/Output : expval, rev_expval
+*/
+static void
+create_expected_data(int iter_flag, int **evt_data,
+					int *expval, int *rev_expval)
+{
+	int epoch;
+	int expected_size;
+	int count;
+	int loop_count;
+	int offset;
+
+	count = 0;
+	switch (iter_flag) {
+	case EVT_ITER_EMBEDDED:
+		print_message("EVT_ITER_EMBEDDED\n");
+	break;
+	case EVT_ITER_VISIBLE:
+		print_message("EVT_ITER_VISIBLE\n");
+		for (epoch = 1; epoch <= NUM_EPOCHS; epoch++) {
```

Comment:
I'd prefer these for loops were in static functions.   If possible, they can be combined into a single static function with parameters to avoid duplication.

Reply:
Each for loop has subtle changes. Let me take a look at it...

Reply:
Done.